reference/replica-operations.md

```diff
@@ -2,27 +2,24 @@
 
-If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompos' itself; all of the previous member replicas will be re-attached to it, if possible, including any that were in the state `CHILD_FAULTED`.  If faulted replicas can be re-connected successfully, then the control plane will attempt to rebuild them directly, rather than seek replacements for them first.  This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If they are successfully re-attached but then continue to encounter I/O failures the rebuild will fail and will not be attempted again.
+If a nexus is cleanly restarted, i.e. the Mayastor pod hosting it restarts gracefully, with the assistance of the control plane it will 'recompose' itself; all of the previous healthy member replicas will be re-attached to it.  If previously faulted replicas are available to be re-connected (`Online`), then the control plane will attempt to reuse and rebuild them directly, rather than seek replacements for them first. This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If the rebuild fails then we follow the above process of removing a `Faulted` replica and adding a new one. On an unclean restart (i.e. the Mayastor pod hosting the nexus crashes or is forcefully deleted) only one healthy replica will be re-attached and all other replicas will eventually be rebuilt.
 
-Once provisioned, the replica count of a volume can be changed by editing its corresponding MayastorVolume \(MSV\) custom resource object.  The value of the `Replica Count` field may be either increased or decreased and the control plane will attempt to reconcile the desired and actual state, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
-
-Whenever attempting to reconcile the replica count of an existing volume, should the control plane fail to locate an eligible pool for a required replica, or should the replica creation call fail, the reconciliation 'loop' for that Volume will cease; the desired vs actual replica count will remain diverged.
+Once provisioned, the replica count of a volume can be changed using the kubectl-mayastor plugin `scale` subcommand.  The value of the `num_replicas` field may be either increased or decreased by one and the control plane will attempt to satisfy the request by creating or destroying a replicas as appropriate, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
```

Comment:
What happens if, for example, I increase by 2?  Do I get an "off by one" error? :-)

Reply:
Something like that:
`Failed to scale volume ec4e66fd-3b33-4439-b504-d49aba53da26. Error error in response: status code '412 Precondition Failed', content: 'RestJsonError { details: "Replica count only allowed to change by a maximum of one at a time", kind: FailedPrecondition }'`

Reply:
Interesting.  So this is currently a limitation of the reconciliation loop for replicas/volume health?  It won't keep adding one replica at a time, waiting for the rebuild to complete and then checking to see whether actual vs desired count is satisfied yet?

Reply:
It's not a limitation of the reconciliation loop - we'll be happy to reconcile whatever the difference between replica spec/state is, it's more a limitation set at the "frontend" level to keep things simple. We should revisit this for the next release.